src-tauri/src/main.rs

```diff
@@ -85,8 +85,10 @@ const LOG_TARGET_WEB: &str = "tari::universe::web";
 
 #[cfg(feature = "release-ci")]
 const APPLICATION_FOLDER_ID: &str = "com.tari.universe";
-#[cfg(not(feature = "release-ci"))]
+#[cfg(feature = "release-ci-beta")]
```

Comment:
```suggestion
#[cfg(and(feature = "release-ci-beta", not(feature="release-ci"))]
```
This might fix the clippy

Reply:
I made a change, but now I think we need to put in a faulty state, if both features ```release-ci``` and ```release-ci-beta```, will give us a consts ```note: `APPLICATION_FOLDER_ID` must be defined only once in the value namespace of this module```?